### ISSUE TITLE
Specifying 2.12.0 version in Github actions waits indefinitely

### ISSUE THREAD

**Issue by dshukertjr** [dart-lang/setup-dart#61]
I am trying to test my code against v2.12.0 of Dart, but specifying the version in Github Actions just waits with the following output until it times out, and was wondering if this is a temporary server issue, or is an issue of this action. 

```bash
Attempt:480 waiting for isolate vm-isolate to check in
Attempt:480 waiting for isolate dartdev to check in
```

```yaml
name: Test

on:
  push:
    branches: [main]
  pull_request:
    branches: [main]

jobs:
  test:
    name: Test / SDK ${{ matrix.sdk }}
    strategy:
      fail-fast: false
      matrix:
        os: [ubuntu-latest]
        node: ["12"]
        sdk: [2.12.0, stable, beta, dev]

    runs-on: ${{ matrix.os }}

    steps:
      - name: Checks-out repo
        uses: actions/checkout@v2

      - name: Setup Dart
        uses: dart-lang/setup-dart@v1
        with:
          sdk: ${{ matrix.sdk }}

      - name: Install dependencies
        run: dart pub get

      - name: dartfmt
        run: dart format lib test -l 80 --set-exit-if-changed

      - name: analyzer
        run: dart analyze --fatal-warnings --fatal-infos .

      - name: Run tests
        run: dart test

```

**Comment by devoncarew** [comment]
The format of the sdks versions looks right. Is this issue intermittent? Does it not happen if you remove 2.12.0. from the list of SDK versions?

I'm not sure what is printing out `waiting for isolate` . Perhaps try commenting out some of the job steps to see if you can bisect the issue?

**Comment by dshukertjr** [comment]
It seems like I had included a package that has minimum SDK of 2.15.0, which is higher than the 2.12.0 that I was trying to test against. Downgrading the package to a version with minimum SDK version of 2.12.0 removed the hanging.
